### ISSUE TITLE
[Accessibility]Narrator's blue cursor cannot move when entering characters into the RichTextBox/ComboBox in RTL environment

### ISSUE THREAD

**Issue by Marie-Shi** [dotnet/winforms#3640]
* .NET Core Version: 
.Net 5.0.100-rc.1.20373.1 from master branch + latest master VS2019 with latest core designer

* Have you experienced this same bug with .NET Framework?: 
No

**More info:**
1.	This issue cannot repro in LTR environment.
2.	When using left/right arrows, it also cannot focus on every letter.

**Steps to reproduce:**
1.	Extract “Private_1588_RTL.zip” and open in in VS, then build and run this application.
[Private_1588_RTL.zip](https://github.com/dotnet/winforms/files/4970937/Private_1588_RTL.zip)
2.	Start Narrator and focus on RichTextBox/ComboBox, input characters into them and observe Narrator blue cursor.

**Actual behavior:**
Narrator's blue cursor cannot move when entering characters into the RichTextBox/ComboBox in RTL environment.
![New_issue1](https://user-images.githubusercontent.com/45864017/88374022-87493b00-cdcb-11ea-963d-e2c80be107a1.gif)

**Expected behavior:**
Narrator's blue cursor can move when entering characters into the RichTextBox/ComboBox in RTL environment. The behavior should be like in LTR.
![Untitled](https://user-images.githubusercontent.com/45864017/88374091-b6f84300-cdcb-11ea-98b8-2b3a6b57f543.png)

**Comment by merriemcgaw** [comment]
We will reexamine this issue if there is an external report that is problematic. Chances are we don't own this UI anyway.
